hardhat.config.ts

```diff
@@ -39,7 +39,9 @@ const {
   POLYGONSCAN_KEY,
   REPORT_GAS = 'false',
   NETWORK_PROVIDER = '',
-  REMOTE_ACCOUNTS = '',
+  GOV_NETWORK_PROVIDER = '',
+  GOV_NETWORK = '',
```

Comment:
Approach recommended by🥈. This allows us to override a specific network's provider with the least amount of env variables. Otherwise, we might need a separate env variable per network.